### PR TITLE
Fix file access to future access list in network module

### PR DESCRIPTION
**Pull request by zarubond**
Hi, I would like to merge my fix for the network module. 
This code enables access to files with future access token instead of local filesystem path.

### CODE REVIEW

**Comment by rozele** [comment]
This is dependent on #1549 being merged.

File: ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs [hunk 1]
```diff
@@ -239,7 +240,15 @@ public override void OnReactInstanceDispose()
             try
             {
 #if WINDOWS_UWP
-                var storageFile = await StorageFile.GetFileFromPathAsync(uri.LocalPath).AsTask().ConfigureAwait(false);
+                StorageFile storageFile;
+                if (FutureAccessList.IsValidURI(uri.ToString()))
+                {
+                    storageFile = await FutureAccessList.GetFileAsync(uri.ToString());
```
Comment (rozele) [inline ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs]: Why use `ConfigureAwait(false)` below but not here?

File: ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs [hunk 2]
```diff
@@ -239,7 +240,15 @@ public override void OnReactInstanceDispose()
             try
             {
 #if WINDOWS_UWP
-                var storageFile = await StorageFile.GetFileFromPathAsync(uri.LocalPath).AsTask().ConfigureAwait(false);
+                StorageFile storageFile;
```
Comment (rozele) [inline ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs]:
```
var storageFile = await FileAccessList.IsValueURI(uri.ToString())
    ? FutureAccessList.GetFileAsync(uri.ToString())
    : StorageFile.GetFileFromPathAsync(uri.LocalPath).AsTask().ConfigureAwait(false)
```

Comment (zarubond) [reply ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs]: Ok, simplified

File: ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs (hunk 1 above)
Comment (zarubond) [reply ReactWindows/ReactNative.Shared/Modules/Network/NetworkingModule.cs]: I have fixed it yesterday, after someone else pointed it to me

**Pull request closed without merging**